controllers/disruption_controller.go

```diff
@@ -424,6 +436,12 @@ func (r *DisruptionReconciler) startInjection(instance *chaosv1beta1.Disruption)
 				continue
 			}
 
+			if disKind == chaostypes.DisruptionKindNodeFailure && instance.Spec.StaticTargeting && injection.InjectionStatus != chaostypes.DisruptionInjectionStatusNotInjected {
```

Comment:
Maybe we could create a small helper method to clarify the intent, something like `shouldSkipNodeFailureReinjection` or similar